h2o-automl/src/main/java/ai/h2o/automl/modeling/DummyStepsProvider.java

```diff
@@ -0,0 +1,143 @@
+package ai.h2o.automl.modeling;
+
+import ai.h2o.automl.*;
+import ai.h2o.automl.events.EventLogEntry;
+import hex.*;
+import hex.genmodel.utils.DistributionFamily;
+import hex.tree.drf.DRF;
+import hex.tree.drf.DRFModel;
+import water.*;
+import water.fvec.Frame;
+
+import static ai.h2o.automl.ModelingStep.ModelStep.DEFAULT_MODEL_TRAINING_WEIGHT;
+
+
+public class DummyStepsProvider
+        implements ModelingStepsProvider<DummyStepsProvider.DummySteps> {
+
+    static class DummySteps extends ModelingSteps {
+
+        private static DRFModel.DRFParameters prepareModelParameters(AutoML aml) {
+            DRFModel.DRFParameters params = new DRFModel.DRFParameters();
+            params._train = aml.getTrainingFrame()._key;
+            if (null != aml.getValidationFrame())
+                params._valid = aml.getValidationFrame()._key;
+
+            AutoMLBuildSpec buildSpec = aml.getBuildSpec();
+            params._response_column = buildSpec.input_spec.response_column;
+            params._ignored_columns = buildSpec.input_spec.ignored_columns;
+            params._fold_column = buildSpec.input_spec.fold_column;
+            params._weights_column = buildSpec.input_spec.weights_column;
+
+            if (buildSpec.input_spec.fold_column == null) {
+                params._nfolds = buildSpec.build_control.nfolds;
+                if (buildSpec.build_control.nfolds > 1) {
+                    params._fold_assignment = Model.Parameters.FoldAssignmentScheme.Modulo;
+                }
+            }
+
+            if (buildSpec.build_control.balance_classes) {
+                params._balance_classes = buildSpec.build_control.balance_classes;
+                params._class_sampling_factors = buildSpec.build_control.class_sampling_factors;
+                params._max_after_balance_size = buildSpec.build_control.max_after_balance_size;
+            }
+
+            params._keep_cross_validation_predictions = true;
+            params._keep_cross_validation_models = buildSpec.build_control.keep_cross_validation_models;
+            params._keep_cross_validation_fold_assignment = buildSpec.build_control.nfolds != 0 && buildSpec.build_control.keep_cross_validation_fold_assignment;
+            params._export_checkpoints_dir = buildSpec.build_control.export_checkpoints_dir;
+
+            return params;
```

Comment:
most of this logic is unnecessary. Actually I would completely remove it, as it overrides the default logic done in `ModelingStep` and would prevent testing the logic in that class.